server/src/item.rs

```diff
@@ -21,6 +21,7 @@ pub struct Item {
     inner: Arc<Mutex<oo7::portal::Item>>,
     // Other attributes
     manager: Arc<Mutex<ServiceManager>>,
+    collection_path: OwnedObjectPath,
```

Comment:
Hmm, that is something I didn't thought of when commenting. Maybe that is not a great idea after all :)

Reply:
Actually, never mind. Given that we need it to construct the Item's path anyways, so let us keep it this way.